.storybook/updated-type.css

```
@@ -18,6 +18,59 @@
     url(/SanaSansLCG05-Variable.ttf) format('truetype');
 }
 
+@font-face {
+  font-family: 'Roboto';
```

Stylelint (17.14.1)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)